src/binary/mod.rs

```diff
@@ -14,7 +14,7 @@ mod nibbles;
 pub mod non_blocking;
 pub mod raw_binary_writer;
 pub mod timestamp;
-mod type_code;
+pub(crate) mod type_code;
```

Comment:
🗺️ This (along with other modules you'll see later) was changed to pub(crate) so it could be referred to from the lazy/binary module as well as the /binary module where it lives.